### PR TITLE
Initialize 2015-scipy directory, add panorama lecture/solution

### PR DESCRIPTION
**Pull request by JDWarner**
This PR adds the `2015-scipy` lecture directory, links to it from package root, and puts `check_env.py` into place. I used Stefan's template from 2015-strata; it looks good, but the lecture links will need revision.

Also added my new, updated panorama lecture with a solved version, copied `solutions_012.ipynb` into the solutions directory, and fixed the original panorama example so it loads only the images it should.

Once this is merged, I'll get to work on #16 for every code-containing notebook in the repo.

### CODE REVIEW

**Comment by JDWarner** [comment]
Ping @stefanv, @blink1073 - this should give us a good foundation on which to build!

**Comment by blink1073** [comment]
@JDWarner, I'll take #16.

File: 2015-scipy/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+scikit-image tutorial at SciPy 2015
+===================================
+
+This directory contains the materials for the ``scikit-image`` tutorial given
+at [SciPy 2015](https://conference.scipy.org/scipy2014/schedule/tutorials/) by
```
Comment (blink1073) [inline 2015-scipy/README.md]: Should link to scipy2015 no?

Comment (JDWarner) [inline 2015-scipy/README.md]: Right on!

**Comment by JDWarner** [comment]
@blink1073 fixed the 2015 tutorials link.

Might want to specifically look for (find in page) `print` within each notebook, so we don't get surprise errors.

**Comment by stefanv** [comment]
Thanks, Josh!

**Pull request merged**